frontend/app/assets/stylesheets/spree/frontend/variables/variables.scss

```diff
@@ -7,6 +7,7 @@ $meganav-font-color: $font-color;
 $meganav-background: $primary-background;
 $footer-font-color: $font-color;
 $footer-background: $primary-background;
+$global-border-style: #e2e2e2;
```

Comment:
I think `$global-border-style` should be used for border attribute. Not only for border color.
`$global-border-style: 1px solid #e2e2e2;`
@AleksandraKaminska wdyt?

Reply:
@pmilewski Good question, but I think it's not necessary